onetimepass/otp.py

```diff
@@ -283,10 +288,29 @@ def import_(ctx: click.Context, file, format_: str):
     db.write(data)
 
 
-# TODO(khanek) Move to separate module
-def get_alias_from_uri(uri: str) -> AliasSchema:
-    uri_parsed = otp_auth_uri.parse(uri)
-    return create_alias_schema(
+@otp.group(
+    context_settings={"help_option_names": ["-h", "--help"]},
+    help="Add the new secret as the specified ALIAS.",
+)
+@click.pass_context
+def add(ctx: click.Context):
+    pass
+
+
+@add.command("uri", help="Add the new secret as the specified ALIAS.")
```

Comment:
I would suggest `help="Add the new secret from URI as the specified ALIAS."`, WDYT?